### PR TITLE
Speed Up selecting/deselecting all rows. And sorting columns that contained a lot of same values

### PR DESCRIPTION
**Pull request by EvgeniiAl**

### CODE REVIEW

File: src/data/sort.js [hunk 1]
```diff
@@ -1,15 +1,9 @@
...
 export default function sortRows(rows, sortColumn, sortDirection) {
-    function comparer(a, b) {
-        if (sortDirection === 'ASC') {
-            return (a[sortColumn] > b[sortColumn]) ? 1 : -1;
-        } else if (sortDirection === 'DESC') {
-            return (a[sortColumn] < b[sortColumn]) ? 1 : -1;
-        }
+    switch (sortDirection) {
+        case 'ASC': return R.sort(R.ascend(R.nth(sortColumn)), rows.slice(0));
+        case 'DESC': return R.sort(R.descend(R.nth(sortColumn)), rows.slice(0));
```
Comment (chriddyp) [inline src/data/sort.js]: very nice ⭐️

**Comment by chriddyp** [comment]
Very nice @EvgeniiAl ! 👏  Thanks for following the Ramda style too 🌟

**Comment by EvgeniiAl** [comment]
@chriddyp Thanks to your great work! I am afraid my fix still have some issue, when user deselect one of  many selected rows, updating selected rows will take a huge amount of time :(

**Comment by chriddyp** [comment]
Woops, I think I merged too early then. I've reverted the change in this PR: https://github.com/plotly/dash-table-experiments/pull/21, feel free to reopen once you fix that last issue that you are seeing :)

**Comment by chriddyp** [comment]
@EvgeniiAl - Just wanted to follow up here. Did this PR slow things down or did it just not fix _all_ of the issues? In other words, are we safe to merge this PR without any performance degradation or did it introduce new issues?

**Comment by EvgeniiAl** [comment]
@chriddyp I hope this can be safely merged.

**Pull request merged**